docs/REST API/rest-api.md

```diff
@@ -4969,20 +4968,18 @@ Updates the specified label. Note: Label queries and platforms are immutable. To
 | id          | integer | path | **Required**. The label's id. |
 | name        | string  | body | The label's name.             |
 | description | string  | body | The label's description.      |
-| hosts       | array   | body | If updating a manual label: the list of host identifiers (`hardware_serial`, `uuid`, `osquery_host_id`, `hostname`, or `name`) the label will apply to. |
```

Comment:
Revised to remove `osquery_host_id` and `name` based on this Slack [thread](https://fleetdm.slack.com/archives/C019WG4GH0A/p1730741789942519?thread_ts=1730469726.071789&cid=C019WG4GH0A).